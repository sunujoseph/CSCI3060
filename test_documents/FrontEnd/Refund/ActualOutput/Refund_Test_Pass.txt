path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Refund\ActualDTF\Refund_Test_Pass.txt
Enter Username: Error: Invalid Input
Error: Not Logged In
Error: Invalid Input
Error: Invalid Input
Error: Invalid Input
Error: Not Logged In
