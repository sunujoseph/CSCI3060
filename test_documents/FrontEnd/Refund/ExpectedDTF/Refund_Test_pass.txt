05 userFS          userSS          012012.34