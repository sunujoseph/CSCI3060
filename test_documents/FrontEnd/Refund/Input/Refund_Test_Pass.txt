login
admin1
refund
item2
userFS
userSS
12012.34
time
