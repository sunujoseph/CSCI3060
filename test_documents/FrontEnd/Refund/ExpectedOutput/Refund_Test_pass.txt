Please log in to the system: 
Please enter your username: 
Enter command: 
Enter Item Name: 
Enter buyer username: 
Enter seller username: 
Enter amount of credit: 
Enter command: